build.sh

```diff
@@ -299,11 +299,11 @@ if [[ "$WATTSI_RESULT" -ne "0" ]]; then
     echo "The correct line numbers for errors in the $HTML_SOURCE/source file are shown below:"
     echo
     cat $HTML_TEMP/wattsi-output.txt | grep -v '^$' # trim blank lines
```

Comment:
> I vaguely recall that back in September when I was first hacking on this, I found that if we always output the error report for the build against the raw source, it always emits errors, because there’s at least one case where there’s a funky dfn value that I guess somehow gets corrected or normalized by wattsi such that it doesn’t emit the error when run against the pre-processed source.
> 
> But I think rightly we should fix that in the source that we can actually run wattsi against the raw source without it reporting errors. Otherwise, those error messages are anyway always going to be tossed into the output if there are other actual fatal errors that cause wattsi to exit non-zero.

So yeah I just now remembered/rediscovered what causes that error message when running wattsi against the raw source: It’s in the source for the [parsing the sort key](https://html.spec.whatwg.org/multipage/tables.html#parsing-the-sort-key) algorithm, which is expressed in a the raw source with a big pseudo-code case-switch statement, which gets converted into markup by the build and as part of that, the build ends up generating a `<dfn data-x="sort parser mode: separator">separator</dfn>` for a term for the `separator` case, which ends up at https://html.spec.whatwg.org/multipage/tables.html#sort-parser-mode:-separator

So, because the `dfn` is not actually in the raw source but the raw source does contain references to it, I don’t know of a workaround that would cause wattsi to not report an “no dfn found” error for that when wattsi is run against the raw source. Maybe @Hixie an suggest something.

In the mean time, with our “run wattsi a second time against the raw source to get the right line numbers” build hack in place, it’s always going to report a `missing <dfn> for topic "sort parser mode: separator"` error there.